src/angular-resizable.js

```diff
@@ -50,16 +53,26 @@ angular.module('angularResizable', [])
 
                 var updateInfo = function(e) {
                     info.width = false; info.height = false;
-                    if(axis == 'x')
+                    if(axis == 'x')						
                         info.width = scope.rFlex ? parseInt(element[0].style.flexBasis) : parseInt(element[0].style.width);
                     else
                         info.height = scope.rFlex ? parseInt(element[0].style.flexBasis) : parseInt(element[0].style.height);
                     info.id = element[0].id;
                     info.evt = e;
-                }
+					info.originalWidth = originalW;
+					info.originalHeight = originalH;
+                };
 
                 var dragging = function(e) {
-                    var offset = axis == 'x' ? start - e.clientX : start - e.clientY;
+                    var offset = (axis == 'x') ? start - e.clientX : start - e.clientY;
+					
+					// bind resize to grid
+					var gridX = scope.rGrid[0] || 1,

```

Comment:
If `r-grid` isn't set on the resizable element, this logs an error. Instead maybe use:

```
var gridX = scope.rGrid ? scope.rGrid[0] : 1,
    gridY = scope.rGrid ? scope.rGrid[1] : 1;
```